wic/src/main/java/edu/asu/diging/wic/core/conceptpower/impl/package-info.java

```diff
@@ -0,0 +1,7 @@
+@javax.xml.bind.annotation.XmlSchema(
+    xmlns = {
+        @javax.xml.bind.annotation.XmlNs(prefix="digitalHPS", namespaceURI="http://www.digitalhps.org/")
+    }
+)
+
+package edu.asu.diging.wic.core.conceptpower.impl;
```

Comment:
is this class necessary since you explicitly have the namespace in each xml element annotation?